src/importexport/musicxml/internal/musicxml/importmxmlpass2.cpp

```diff
@@ -6228,12 +6265,15 @@ static void addTie(const Notation& notation, Score* score, Note* note, const tra
         tie->setStartNote(note);
         tie->setTrack(track);
 
-        if (orientation == "over") {
+        const QColor color { notation.attribute("color") };
+        if (color.isValid()) {
+            tie->setColor(color);
+        }
+
+        if (orientation == "over" || placement == "above") {
```

Comment:
Here the order is the same as in my #17842, but different from your slur handling

Reply:
This too should be dependant of
```cpp
 if (configuration()->musicxmlImportLayout()) {
```